core/src/main/java/nu/yona/server/exceptions/ResourceBasedException.java

```diff
@@ -73,6 +101,23 @@ protected ResourceBasedException(HttpStatus statusCode, Throwable t, String mess
 		this.statusCode = statusCode;
 	}
 
+	@Override
+	public String getMessage()
+	{
+		return getLocalizedMessage();
+	}
+
+	@Override
+	public String getLocalizedMessage()

```

Comment:
Please update the `GlobalExceptionMapping` class to use this method instead of directly calling `Translator`.